deploy/docker/gaiahub.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+TASK=$1
```

Comment:
It's best to quote arguments (`TASK="$1"`).  Can you run `shellcheck` on this script and correct any warnings it reports?